doc/release-notes-25355.md

```diff
@@ -0,0 +1,5 @@
+New settings
+------------
+
+`-i2ptransientout` has been added to generate a new I2P address for each
```

Comment:
It seems important to first announce the change of default behavior (maybe in the Notable Changes or P2P section): With I2P connections, a new, transient address is announced to each outbound peer by default. And then that a config option is added to allow toggling back to the previous behavior.

```suggestion
A new `-i2ptransientout` configuration option has been added...
```

Reply:
Done.